homework/index.js

```diff
@@ -30,14 +30,103 @@
     return elem;
   }
 
+  function clearContainer(container) {
+    while (container.firstChild) {
+      container.removeChild(container.firstChild);
+    }
+  }
+
+  function renderError(err) {
+    const listContainer = document.getElementById('repo_information_table-container');
+    clearContainer(listContainer);
+    const root = document.getElementById('root');
+    createAndAppend('div', root, { text: err.message, class: 'alert alert-error' });
+  }
+
+  function addRow(tbody, label, value = '') {
+    const row = createAndAppend('tr', tbody);
+    createAndAppend('td', row, { text: `${label} :`, class: 'label' });
+    createAndAppend('td', row, { text: value });
+    return row;
+  }
+
+  function createInfoList(repository, container) {
+    clearContainer(container);
+    const tbody = createAndAppend('tbody', container);
```

Comment:
You are missing a `table` tag. A `tbody` should be the child of a `table` tag, it cannot exist on itself.